wiki/People/en.md

```diff
@@ -4,5 +4,7 @@ Index page for articles about people or groups.
 
 - [Community Contributors](Community_Contributors)
 - [The Team](The_Team)
+- [Tournament Committee](Tournament_Committee)
 - [peppy](peppy)
+- [Performance Points Committee](Performance_Points_Committee)
```

Comment:
didn't you want to have the list sorted?

Reply:
woops